ui/app/common/pnc-client/resources/BuildConfiguration.js

```diff
@@ -89,7 +89,7 @@
       });
 
       resource.getSupportedGenericParameters = function() {
-        return $http.get(restConfig.getPncUrl() + '/build-configurations/supported-generic-parameters').then(function (r) {
+        return $http.get(restConfig.getPncRestUrl() + '/build-configs/supported-parameters').then(function (r) {
```

Comment:
`BuildConfiguration.js` is deprecated as `BuildConfigResource.js` is already available, the correct solution should implement missing methods there as start using `BuildConfigResource` factory method.